tests/end_to_end_test/snapshot_migrate_test.go

```diff
@@ -52,7 +52,7 @@ func (s *formatSpecificTestSuite) TestSnapshotMigrate(t *testing.T) {
 
 	dstenv.RunAndExpectSuccess(t, "repo", "create", "filesystem", "--path", dstenv.RepoDir)
 
-	dstenv.RunAndExpectSuccess(t, "snapshot", "migrate", "--source-config", filepath.Join(e.ConfigDir, ".kopia.config"), "--all", "--parallel=5", "--overwrite-policies")
+	dstenv.RunAndExpectSuccess(t, "snapshot", "migrate", "--read-ahead-mb=30", "--source-config", filepath.Join(e.ConfigDir, ".kopia.config"), "--all", "--parallel=5", "--overwrite-policies")
```

Comment:
It'd be great to add a test that verifies the max go-routine explosion of prefetches in the worst case.

Reply:
will do.